src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -317,6 +341,8 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 
 				if (nativeCrashDetected) {
 					Console.WriteLine ($"Native crash detected! Running the build for {projectOrSolution} again.");
+					File.Move(processLog, processLog + ".bak");
+					File.Delete(processLog);
```

Comment:
How will this actually delete anything? The previous line ensures that `processLog` will no longer exist, unless the directory that `processLog` is in doesn't allow renaming...in which case, what are the odds that the `File.Delete()` will work either?